content/en/product_analytics/heatmaps.md

```diff
@@ -87,7 +87,7 @@ The panel to the left of the Scroll map provides high-level insights with direct
 
 ## Backgrounds
 
-A background is a snapshot of a Session Replay. Each heatmap fetches the 20 backgrounds that triggered the most actions during a given session. You can use the **Choose Background** button to select a particular background for your heatmap.
+A background is a snapshot of a Session Replay. Each heatmap fetches the 20 backgrounds that triggered the most actions during a given session. Changing the background shows different results depending on the background selected. You can use the **Choose Background** button to select a particular background for your heatmap.
```

Comment:
```suggestion
A background is a snapshot of a Session Replay. Each heat map fetches the 20 backgrounds that triggered the most actions during a given session. Changing the background shows different results depending on the background selected. You can use the **Choose Background** button to select a particular background for your heat map.
```
I know this was already here, but vale is flagging it. Not sure if it is correct or not. 🤔

Reply:
Thanks @brett0000FF , I know this is tricky but in this case the feature is called Heatmaps (1 word). Maybe we can bring this up to the style council next time.